apps/system/test/marionette/video_test.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
```

Comment:
This file is part of the system app? Shouldn't it be part of the search app?
The search app already has a browser test there? http://mxr.mozilla.org/gaia/source/apps/search/test/marionette/browser_test.js
Also, perhaps this file should be called browser_video_test.js, to make sure that it's not mistaken for a video_player test.